package.json

```diff
@@ -29,11 +29,11 @@
     "nconf": "^0.10.0",
     "nssocket": "^0.6.0",
     "object-assign": "^4.1.1",
-    "optimist": "^0.6.1",
```

Comment:
Can you remove reference to `optimist` from `cli.js` as well?

Reply:
Done! Sorry for the slow response.